src/psse_export.jl

```diff
@@ -0,0 +1,599 @@
+        end  # TODO fix units
+        QT = PSY.get_reactive_power_limits(generator).max
+        isfinite(QT) || (QT = PSSE_DEFAULT)  # Catch Inf, etc.
+        QB = PSY.get_reactive_power_limits(generator).min
+        isfinite(QB) || (QB = PSSE_DEFAULT)
+        VS = PSY.get_magnitude(PSY.get_bus(generator))  # TODO is this correct? Should this be `get_internal_voltage` for `PSY.Source`?
+        IREG = get(PSY.get_ext(generator), "IREG", PSSE_DEFAULT)
+        MBASE = PSY.get_base_power(generator)
+        ZR, ZX = PSSE_DEFAULT, PSSE_DEFAULT
+        RT, XT = PSSE_DEFAULT, PSSE_DEFAULT  # TODO?
+        GTAP = PSSE_DEFAULT
+        STAT = PSY.get_available(generator) ? 1 : 0
+        RMPCT = PSSE_DEFAULT
+        # TODO maybe have a better default here
+        PT = try
+            PSY.get_active_power_limits(generator).max
+        catch
+            PSSE_DEFAULT
+        end
+        PB = try
+            PSY.get_active_power_limits(generator).min
+        catch
+            PSSE_DEFAULT
+        end
+        O1, O2, O3, O4 = PSSE_DEFAULT, PSSE_DEFAULT, PSSE_DEFAULT, PSSE_DEFAULT
+        F1, F2, F3, F4 = PSSE_DEFAULT, PSSE_DEFAULT, PSSE_DEFAULT, PSSE_DEFAULT
+        WMOD = get(PSY.get_ext(generator), "WMOD", PSSE_DEFAULT)
+        WPF = get(PSY.get_ext(generator), "WPF", PSSE_DEFAULT)
+        joinln(
+            io,
+            [I, ID, PG, QG, QT, QB, VS, IREG, MBASE, ZR, ZX, RT, XT, GTAP, STAT,
+                RMPCT, PT, PB, O1, F1, O2, F2, O3, F3, O4, F4, WMOD, WPF],
+        )
+    end
+    println(io, "0")  # End of section
+    md["generator_name_mapping"] = generator_name_mapping  # TODO reshape to be better for import
+end
+
+"""
+WRITTEN TO SPEC: PSS/E 33.3 POM 5.2.1 Fixed Bus Shunt Data
+"""
+function _write_non_transformer_branch_data(
+    io::IO,
+    md::AbstractDict,
+    sys::System,
+    psse_version::Symbol,
+)
+    check_33(psse_version)
+
+    # TODO can/should we be more general than `Line`?
+    branches = sort!(collect(PSY.get_components(PSY.Line, sys));
+        by = branch ->
+            PSY.get_number.((PSY.get_from_bus(branch), PSY.get_to_bus(branch))))
+    branch_to_bus_numbers(branch) =
+        (PSY.get_number.((PSY.get_from_bus(branch), PSY.get_to_bus(branch))))
+    branch_name_mapping =
+        create_component_ids(
+            PSY.get_name.(branches),
+            branch_to_bus_numbers.(branches);
+            singles_to_1 = false,
+        )
+
+    for branch in branches
+        I = md["bus_number_mapping"][PSY.get_number(PSY.get_from_bus(branch))]
+        J = md["bus_number_mapping"][PSY.get_number(PSY.get_to_bus(branch))]
+        CKT = branch_name_mapping[((I, J), PSY.get_name(branch))]
+        @assert !(first(CKT) in ['&', '@', '*'])  # Characters with a special meaning in this context
+        CKT = _psse_quote_string(CKT)
+        R = PSY.get_r(branch)
+        X = PSY.get_x(branch)
+        B = 0.0  # TODO iron out the details of B vs. BI, BJ
```

Comment:
The summary about this issue is that the line model in PSSe uses a data model with 2 shunt elements per side and this creates a mess in terms of writing back the data since PSY supports only one imaginary shunt per side. 


![image](https://github.com/user-attachments/assets/743e3abb-d9a4-4251-9d96-b3708bca374d)